app/helpers/json_presenter_helper.rb

```diff
@@ -0,0 +1,7 @@
+module JsonPresenterHelper
+
+    def render_response(response)
+        render json: response
+    end
+end
+
```

Comment:
No changes requested:
I like that you utilized a module to extend your ApplicationController class. On the flipside though, sometimes when youre reading legacy code in large apps you'll be like "Where is this method coming from 😱 !!" and it will most likely be a module. So that is the tradeoff of not using a class and saying `JsonPresenterHelper.render_response("hello")`. Again no changes requested, I wanted to let you know about the tradeoffs of a module.

Reply:
Thank you for all your useful feedback!